### PR TITLE
Feat/travis cleanup speedup

### PR DESCRIPTION
**Pull request by czervenka**
do not merge yet please

### CODE REVIEW

**Comment by xaralis** [comment]
> do not merge yet please

If you do not want this to be merged, it's always good idea to make this a draft PR.

File: .travis.yml [hunk 1]
```diff
@@ -9,37 +9,56 @@ addons:
...
     - stage: Run tests
       language: node_js
-      name: frontend E2E cypress tests
+      name: Compose test environment and run e2e tests
+
+    # run cypress tests
       node_js: 10
       cache:
         directories:
-          - node_modules
```
Comment (xaralis) [inline .travis.yml:22]: Can you please explain why this was removed?

Comment (czervenka) [reply .travis.yml:22]: caching node_modules caches the state of last app installation while we need a fresh environment to do tests. `.npm` directory, on the other hand, is just a cache.

**Comment by czervenka** [comment]
> > do not merge yet please
> If you do not want this to be merged, it's always good idea to make this a draft PR.
Agree, I could not find how to do that.

**Pull request merged**